scraper/src/strategies/default_strategy.py

```diff
@@ -177,7 +177,12 @@ def get_records_from_dom(self, current_page_url=None):
                         record[name] = content
 
                     if name == "version":
-                        record[name] = str(content)
+                        version = str(content)
+                        # When version is a comma-separated tokens
+                        if ',' in version:
```

Comment:
We shouldn't need to check this conditionally because `split` returns an array no matter if the pattern matches:

```python
>>> "1.0.0,latest".split(",")
['1.0.0', 'latest']
>>> "1.0.0".split(",")
['1.0.0']
```

Reply:
Fair point. Resolved